src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -305,7 +285,8 @@ class DiscoveryItem(BaseModel):
 
 
 class FieldItem(BaseModel):
-    """Reference to a field known to SMDA."""
+    """The ``masterdata.smda.field`` block contains a list of fields
```

Comment:
same here

Reply:
Commenting without context here, but if this is referring to a specific field _item_, it is not a list. However, the `masterdasta.smda.field` block is a list (of items where each item represents a field)